admin/app/controllers/admin/CommercialController.scala

```diff
@@ -126,7 +126,7 @@ class CommercialController(
       val lineItems: Seq[GuLineItem] = Store.getDfpLineItemsReport().lineItems filter (_.orderId.toString == orderId)
 
       Cached(5.minutes) {
-        JsonComponent(Json.toJson(lineItems))
+        JsonComponent.fromWritable(lineItems)
```

Comment:
Two things are happening here:

1. We're moving from the ambiguous `apply()` method invocation (ie `JsonComponent()`) to the unambiguous `JsonComponent.fromWritable()` invocation - it's still the same method implementation we're calling, it's just the method is now explicitly named.
2. We're removing a superfluous call to `Json.toJson()` - that was always superfluous, it's just clearer to see now when we look at the `JsonComponent.fromWritable()` method that it also does a call to `Json.toJson()`, so there's no point in doing it twice.